ubyssey/templates.py

```diff
@@ -44,11 +44,12 @@ class FullWidth(Template):
 class Guide(Template):
     id = 'guide-to-ubc'
     name = 'Guide to UBC'
-
+    banner_quote = CharField('Banner-quote')
```

Comment:
```suggestion
    banner_quote = CharField('Banner Quote')
```

The argument passed to `CharField` is the field name that is displayed to the user in the admin panel, so best to format it for readability.